modules/vector-search/pages/pre-filtering-vector-search.adoc

```diff
@@ -0,0 +1,64 @@
+= Pre-filtering Vector Searches
+:page-topic-type: guide
+:page-ui-name: {ui-name}
+:description: You can specify filters as part of a vector search statement which will restrict the documents searched during the query.
+
+[abstract]
+{description}
+
+== About Pre-filtering
+
+Using pre-filtering as part of your vector search offers two key advantages:
+
+[horizontal]
+Enhanced precision and relevance::
+allows searches to be narrowed down to results based on specific criteria (eg., organization, date/time ranges, geospatial locations.)
+
+Performance optimization::
+by reducing the search space before executing the queries, you can improve query execution time and reduce computational overhead.
+
+== Prerequisites
+
+* You have the Search Service enabled on a node in your database.
+For more information about how to deploy a new node and Services on your database, see xref:server:manage:manage-nodes/node-management-overview.adoc[].
+
+* You have a bucket with scopes and collections in your database.
+For more information about how to create a bucket, see xref:server:manage:manage-buckets/create-bucket.adoc[].
+
+* Your user account has the *Search Admin* or *Search Reader* role.
+
+* You installed the Couchbase command-line tool (CLI).
+
+* You have the hostname or IP address for the node in your database where you're running the Search Service.
+For more information about where to find the IP address for your node, see xref:server:manage:manage-nodes/list-cluster-nodes.adoc[].
+
+* You have created a Vector Search index.
++
+For more information about how to create a Vector Search index, see xref:create-vector-search-index-ui.adoc[] or xref:create-vector-search-index-rest-api.adoc[].
++
+[TIP]
+--
+include::partial$download-sample-partial.adoc[]
+
+For the best results, consider using the sample Vector Search index from xref:create-vector-search-index-ui.adoc#example[Create a Vector Search Index with the {page-ui-name}] or xref:create-vector-search-index-rest-api.adoc#example[Create a Vector Search Index with the REST API and curl/HTTP].
+--
+
+== Procedure
+
+To run a pre-filtered vector search with the REST API:
+
+. In your command-line tool, enter a `curl` command with the `XPOST` verb.
+. Set your header content to include `"Content-Type: application/json"`.
+. Enter your username, password, and the Search Service endpoint on port `8094` with the name of the Vector Search index you want to query:
+
+
+=== Example
+
+In the following example, you will extend a search query to employ a pre-filter to narrow the set of documents that the query will search over.
+
+[source, console]
+----
+include::example$run-pre-filtered-vector-search-rest-api.sh[]
+----
```

Comment:
Can we add in a brief explanation of what the actual filter is that's applied in this example? I'm assuming searching for "navy" in the "color" field, but

Reply:
Right, I think I've got it.